chpl/chpl-resources/src/main/resources/email.properties

```diff
@@ -22,7 +22,6 @@ questionableActivityNoDataEmailBody=<p>No questionable activity was found betwee
 #pending change request email properties
 pendingChangeRequestReportFilename=pending-change-request
 pendingChangeRequestEmailSubject=Pending Change Request Report
-pendingChangeRequestEmailAcbSubject=ONC-ACB Pending Change Request Report
```

Comment:
There were four duplicate reports; did only one of them have values in this property file?

Reply:
This was the only report that used a different subject for the two emails.